plugins/oauth/hooks.js

```diff
@@ -62,6 +64,21 @@ module.exports = Object.assign({}, StoreMixin, {
     }]);
   },
 
+  dcosInstallCommand(instructions) {
+    let steps = instructions.split('&&');
+    steps.splice(-1, 0, ' \n dcos auth login ');
```

Comment:
Could we get a comment here? something like: "Add login as the step second to last, before printing command options"